### ISSUE TITLE
Make the site https?

### ISSUE THREAD

**Issue by karanjthakkar** [vaidehijoshi/better-code-reviews#5]
Hey, wondering if you want to get this running on https? I'd love to help if you do :)

**Comment by vaidehijoshi** [comment]
Sure! I haven't actually ever done that before; I'd love your help! 😺

**Comment by karanjthakkar** [comment]
Great! I'm assuming the site is hosted on Github. If so, I wrote this article to help get started: https://medium.freecodecamp.com/an-illustrated-guide-for-setting-up-your-website-using-github-cloudflare-5a7a11ca9465 Happy to help you run through it whenever you're free.

**Comment by vaidehijoshi** [comment]
Hm, it's a middleman site on Heroku; I'm hoping the process of setting up isn't _too_ different from Github pages?

**Comment by karanjthakkar** [comment]
Nope. It shouldn't be very different. Just skip through the initial Github pages setup. Everything else should still be valid. 👍

**Comment by vaidehijoshi** [comment]
Great, I'll try to get around to this in the next week or two!

**Comment by karanjthakkar** [comment]
Fantastic! 👯‍♂️ Reach out to me on Twitter anytime if you run into something that doesn't make sense 🙂
